packages/functional-tests/tests/signUp/signUpWithCode.spec.ts

```diff
@@ -32,6 +32,9 @@ test.describe('severity-1 #smoke', () => {
         credentials.sessionToken
       );
       await login.waitForPasswordHeader();
+
+      // for linting...
+      expect(true).toBeTruthy();
```

Comment:
Looks like `waitForPasswordHeader` returns the header, so you might be able to do 
```
await expect(settings.passwordHeader).toBeVisible();
```

so its asserting something related to the test.